app/controllers/jobs_controller.rb

```diff
@@ -39,6 +39,15 @@ def run
     end
   end
 
+  def retry_queued
+    @jobs = Delayed::Job.where('failed_at IS NULL AND attempts > 0 AND locked_at IS NULL ').update_all(run_at: Time.now, attempts: 0)
```

Comment:
See delayed_job.rb in initializers/.  You can use those scopes here

```ruby
@jobs = Delayed::Job.awaiting_retry
```

But as I look at those scopes, I think they're not completely correct.  We'd want to ensure that locked_at is null in the awaiting_retry one, for example, to make sure it's not currently being worked on.